gr-utils/blocktool/core/parseheader_generic.py

```diff
@@ -75,10 +75,9 @@ def initialize(self):
 
         if type(self.target_file) == list:
             self.module = self.target_file
-            for dirs in self.target_file:
-                if not os.path.basename(self.module).startswith(Constants.GR):
```

Comment:
Shouldn't `self.module` be changed to `dirs` instead? I am not aware about basename for a list.

Reply:
And if that is the case, please update `dirs` to `dir` instead.

Reply:
I would need to step through to remember what was happening here.  You may be correct - this is just replicating the existing behavior